### PR TITLE
508 profile newprofile

### PR DESCRIPTION
**Pull request by Atticus29**
# Description

This PR fixes the issues found by TotalValidatorPro on the /profile/newprofile.php page.

- Add DOCTYPE
- Add lang attribute
- Move fieldset element inside form element
- Remove the table element and all table-related child elements. Replace them with section elements styled in a way that makes the inputs and input labels behave in a grid-like way with different screensizes.

## Before:

Screen Shot 2023-07-17 at 4 57 51 PM

## After:

Screen Shot 2023-07-18 at 5 28 38 PM

## After, small screen

Screen Shot 2023-07-18 at 5 29 11 PM

# Pull Request Checklist:

- [x] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [ ] Hotfixes should be merged into both the `Development` and `master` branches (at the same time).
    - N/A
- [x] All new text is preferrably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages)
- [x] There are no linter errors
- [x] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [x] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [ ] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
    - N/A
- [ ] Comment which GitHub issue(s), if any does this PR address
    - https://github.com/orgs/BioKIC/projects/6/views/10?pane=issue&itemId=31321457

# TODO

- [ ] It is the code author's responsibility to merge their own pull request after it has been approved
- [ ] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If the dev team has agreed that this PR represents the last PR going into the Development branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place

Thanks for contributing and keeping it clean!

### CODE REVIEW

**Review by themerekat — APPROVED** [review]
Looks great!

**Review by MuchQuak — APPROVED** [review]
Simple Change lgtm once merge conflicts are solved

**Pull request merged**
